### PR TITLE
[docstring] Fix docstring for ErnieConfig, ErnieMConfig

### PR DESCRIPTION
**Pull request by Sparty**
# What does this PR do?

Fixes https://github.com/huggingface/transformers/issues/26638

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by ArthurZucker** [comment]
Feel free to ping @ydshieh when this is ready for a review!

**Comment by ydshieh** [comment]
@Sparty Once ready for a review, could you remove the Draft mode. Thanks

**Comment by Sparty** [comment]
I removed the draft mode. Thank you @ydshieh and @ArthurZucker

File: src/transformers/models/ernie_m/configuration_ernie_m.py [hunk 1]
```diff
@@ -61,19 +61,22 @@ class ErnieMConfig(PretrainedConfig):
...
         layer_norm_eps (`float`, *optional*, defaults to 1e-05):
             The epsilon used by the layer normalization layers.
         classifier_dropout (`float`, *optional*):
             The dropout ratio for the classification head.
-        initializer_range (`float`, *optional*, defaults to 0.02):
-            The standard deviation of the normal initializer for initializing all weight matrices.
-        pad_token_id(`int`, *optional*, defaults to 1):
-            The index of padding token in the token vocabulary.
+        is_decoder (`bool`, *optional*, defaults to `False`):
+             Whether the model is used as decoder or not (in which case it's used as an encoder).
```
Comment (ydshieh) [inline src/transformers/models/ernie_m/configuration_ernie_m.py:77]: We can remove this from docstring and from the arugment, as it is default `False` in the parent class `PretrainedConfig`.

**Review by ydshieh — APPROVED** [review]
Thanks! One tiny comment and we are ready go go!

**Comment by ArthurZucker** [comment]
I'd want to merge, but we need this to be rebased @Sparty if you can run something like `git pull upstream main` and push 🤗

**Comment by Sparty** [comment]
I have rebased. @ydshieh @ArthurZucker Could you merge this?

**Comment by ydshieh** [comment]
Thank you 🤗

**Pull request merged**